django/db/backends/sqlite3/introspection.py

```diff
@@ -413,7 +413,7 @@ def get_constraints(self, cursor, table_name):
                     }
                 constraints[index]['columns'].append(column)
             # Add type and column orders for indexes
-            if constraints[index]['index'] and not constraints[index]['unique']:
+            if constraints[index]['index']:
```

Comment:
This is a separate issue. I've created a new ticket-32453, added tests, and PR #14014.

Reply:
Thanks, I'll rebase after it's merged.